tasks/collect_logs.yml

```diff
@@ -0,0 +1,55 @@
+- name: Ensure local logs dir exists
+  connection: local
+  file:
+    state: "{{ item }}"
+    path: "{{ rhv_he_deploy_log_dir }}"
+  # remove then recreate log dir
+  with_items:
+    - absent
+    - directory
+- name: Check if remote logs dir exists
+  stat:
+    path: /var/log/ovirt-hosted-engine-setup
+  register: rhv_he_deploy_remote_log_dir
+- name: "Write log to {{ rhv_he_deploy_log_dir }} on controller"
+  connection: local
+  copy:
+    content: "{{ item.value }}"
+    # e.g. hosted-engine-deploy-stdout.YYYY-MM-DDTHH:MM:SSZ.log
+    dest: "{{ playbook_dir }}/logs/hosted-engine-deploy-{{ item.key}}.\
+           {{ ansible_date_time.iso8601 }}.log"
+  with_dict:
+    stdout: "{{ rhv_he_deploy_deployment.stdout }}"
+    stderr: "{{ rhv_he_deploy_deployment.stderr }}"
+  when: rhv_he_deploy_deployment is defined and
+        rhv_he_deploy_deployment is not skipped
+- block:
+    - name: "Archive engine host log files"
+      # Ideally, this would be a recursive fetch, but at the time of
+      # writing, the fetch module does not yet have recursive
+      # superpowers of recursion. Instead, make and fetch an archive.
+      archive:
+        path: /var/log/ovirt-hosted-engine-setup
+        dest: /var/log/ovirt-hosted-engine-setup/archive.tar.gz
```

Comment:
We should probably include some unique characteristic for `archive.tar.gz` so that multiple hosts can be installed in parallel. Likewise we might want to prefix the files into directories based on the same unique characteristic.

Reply:
Installing to multiple hosts is a really bad idea, but as with other OASIS roles and our attitude in general I agree that we should generally not decide what's best for a deployment by coding in static stuff like this. I'll fix it up.

Reply:
I realize that this engine installs standalone, but is there any reason I wouldn't be able to install 4 standalone RHV servers with this role at once? If so, we might want to document it in README.md that this should be run separately for each host.

Reply:
The readme already says that it's intended to be run on a single host. What would you recommend doing in addition to that?

Reply:
Locally, the log dir should now be unique per host in the playbook.